upgrade-notes/backend_20250707_082537.md

```diff
@@ -0,0 +1,4 @@
+#### Implement Context system ([#4064](https://github.com/shopsys/shopsys/pull/4064))
+
+- `Shopsys\FrameworkBundle\Model\Administration\AdministrationFacade` was deleted.
```

Comment:
it would be nice to tell the developers, what should they use instead :wink:

Reply:
Edited